src/main/java/core/basesyntax/servise/impl/ReportWriterImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.servise.impl;
+
+import core.basesyntax.servise.ReportWriter;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReportWriterImpl implements ReportWriter {
+    @Override
+    public void write(String report) {
+        File file = new File("src/main/resources/report.csv");
+        try {
+            BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(file));
```

Comment:
use try-with-resources - your resource now won't be closed if there is an exception